ui/app/templates/components/calendar-widget.hbs

```diff
@@ -1,9 +1,5 @@
 <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
-  <D.Trigger
-    data-test-popup-menu-trigger="true"
```

Comment:
This test selector was causing test failures since there were other selectors named the same in the view (ember-basic-dropdown uses the same selector I believe)